src/Roave/SecurityAdvisories/AdvisorySources/GetAdvisoriesFromGithubApi.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/*
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software consists of voluntary contributions made by many individuals
+ * and is licensed under the MIT license.
+ */
+
+declare(strict_types=1);
+
+namespace Roave\SecurityAdvisories\AdvisorySources;
+
+use Generator;
+use Nyholm\Psr7\Request;
+use Psr\Http\Client\ClientExceptionInterface;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestInterface;
+use Roave\SecurityAdvisories\Advisory;
+use Safe\Exceptions\JsonException;
+use Safe\Exceptions\StringsException;
+use stdClass;
+use Webmozart\Assert\Assert;
+use function array_map;
+use function array_merge;
+use function end;
+use function explode;
+use function Safe\json_decode;
+use function Safe\json_encode;
+use function Safe\sprintf;
+
+final class GetAdvisoriesFromGithubApi implements GetAdvisories
+{
+    private const GRAPHQL_QUERY = 'query {
```

Comment:
This would be a bit better if using `<<<'GRAPHQL'`, since indentation would become a bit simpler